### DISCUSSION TITLE
IR Watchdog : Alert/Beep Settings -- Feedback Needed

### DISCUSSION THREAD

**Discussion by nelsonii** [ATMakersOrg/EyeAlert#15] [Q&A]
My initial alarm setup is as follows:

Level 1 : IR lost for 30 seconds. Beep every 10 seconds.
Level 2 : IR lost for 60 seconds. Beep every 5 seconds.
Level 3 : IR lost for 180 seconds. Beep every second.

Do you think these timings work? I'm trying to have enough time to handle repositioning / caregiving without being too annoying.

**Comment by nelsonii** [comment]
Here's a link to a sample video. It's boring, but you get the idea.

https://www.youtube.com/watch?v=2koLdCA7Wcg

**Comment by NachumLehman** [comment]
We need to hear from everyone about this as it will impact on caregivers and we always need to be very mindful of them. Sometimes a device needs to be turned off for a while, sometimes it needs to be restarted and often times, it's moved away from the user for them to give medical care. My initial impression would be to have the first alarm sound after 5 minutes and more significant alarms at 15 and 30 minutes. The light indicator will be extremely useful for a caregiver in the room to know it's off even before the alarm sounds.

**Reply by ImproveAbility** [reply]
Agree with Nachum - that is where my thought about attaching it to the device would be useful - no need to worry about when the device moves for caregiving.

**Comment by ImproveAbility** [comment]
Ron, you are doing a great job!  Thank you for all your hard work on this.  I think we need to be mindful about where this would be positioned.  I am no IR expert, but I am thinking the closer to the device the less interference you would have with it bouncing off of other items.  I don't know what the receiver looks like, but I wonder if you could actually attach it to the IR window on the device.  I know from experience there are some spots that you can cover up on the eye gaze bar and it still works.  Other stops stop the tracking dead.  How feasible would it be to attach it directly to the IR window?
